### PR TITLE
Testing lab trace preview

### PR DESCRIPTION
**Pull request by GRA0007**
image
image

### CODE REVIEW

**Review by tdib — APPROVED** [review]
LGTM!

File: frontend/src/components/TracePreview/TracePreview.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <StyledInitialArrow viewBox="0 0 14 32">
+    <polygon points="14,16 2,0 2,32" />
+  </StyledInitialArrow>
+)
+
+const Transition = ({ error }) => (
+  <StyledTransition viewBox="0 0 25 32" $error={error}>
+    {!error && <path d="M0 16 L24 16 m-7 -7 l7 7 l-7 7" />}
+    {error && <path d="M0 16 L22 16 m-7 -7 l7 7 l-7 7 M24 8 l0 16" />}
+  </StyledTransition>
```
Comment (giraugh) [inline frontend/src/components/TracePreview/TracePreview.js:13]: manual svg path? :eyes:

**Review by giraugh — COMMENTED** [review]
LGTM +1

File: frontend/src/components/TracePreview/TracePreview.js (hunk 1 above)
Comment (GRA0007) [reply frontend/src/components/TracePreview/TracePreview.js:13]: is that illegal

**Pull request merged**